obci/interfaces/bci/budzik/experimental/budzik_prototype.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from obci.configs import settings
+from obci.interfaces.bci.analysis_master import AnalysisMaster
+from obci.interfaces.bci.budzik.experimental.dummy_classifier import DummyClassifier
+from obci.analysis.buffers.auto_ring_buffer import AutoRingBuffer
+
+
+class BudzikPrototype(AnalysisMaster):
```

Comment:
z tego co ja widzę to add_result kolekcjonuje prawdopodobieństwa a nie "decyzje"... standardowo peer ma  metodę self.send_dec(self,dec) która wywoływana jest przez analizę "podpiętą do buffora" jak ta podjęła decyzje